docs/spec/api.md

```diff
@@ -120,6 +120,13 @@ indicating what is different. Optionally, we may start marking parts of the
 specification to correspond with the versions enumerated here.
 
 <dl>
+  <dt>2.0.3</dt>
+	<dd>
+        <ul>
+	     	<li>Implement the delete API for blobs and manifests by removing link files.</li>
```

Comment:
This specification doesn't "implement" anything. Maybe we "specified" deletion for blobs/manifests?